src/components/account/IFXAccount.js

```diff
@@ -1,6 +1,24 @@
 import IFXItemBase from '@/components/item/IFXItemBase'
 
-export default class Account extends IFXItemBase {
+export class UserProductAccount extends IFXItemBase {
+  get user() {
+    return this.data.user
+  }
+
+  get product() {
+    return this.data.product
+  }
+
+  get percent() {
+    return this.data.percent
+  }
+
+  get isValid() {
+    return this.data.is_valid
+  }
+}
+
```

Comment:
I take it there should be no setters for these as they will never be set in the front end?

Reply:
The way they are currently used is read only.  May change some day, but not now.